src/Input/Textarea.js

```diff
@@ -190,6 +190,7 @@ class Textarea extends Component {
           onChange={this.handleChange}
           defaultValue={defaultValue}
           value={value}
+          disabled={disabled}
```

Comment:
You mean removing it from the destructured `this.props` and letting it go into the `other` rest spreader?
Could do that too, I did it this way because that's how it's done in `./Input.js`.

If you'd prefer that, let me know and I'll change it.